criu/cgroup.c

```diff
@@ -732,6 +732,9 @@ int dump_task_cgroup(struct pstree_item *item, u32 *cg_id, struct parasite_dump_
 	unsigned int n_ctls = 0;
 	struct cg_set *cs;
 
+	if (opts.uid)
```

Comment:
Maybe add a wrapper like `has_caps_*()` for this test as well?

Reply:
What do you exactly mean? A capability check for CGROUP or a functions which checks the for uid 0?

Reply:
I meant to have a wrapper function for checking `opts.uid`. E.g. `god_mode()` or more modest `are_we_root()` :)
`

Reply:
There is already a function `is_root_user()` but it does a geteuid() every time. Using `opts.uid` I do not need to do a syscall every time. Not sure how important that is. I also looked into putting `if (opts.uid)` into a function but it felt a bit strange to replace `if (opts.uid)` with a function. Seems unnecessary.

Reply:
Well, my feeling was that `opts.uid` is kinda an instruction from user and a function like `is_root_user()` says what is our
state. 
But I won't insist, we can use `opts.uid`